### PR TITLE
fix: cursor blink lags when pointer down at text range

### PR DESCRIPTION
**Pull request by lumixraku**
close #xxx

## Pull Request Checklist

- [ ] Related tickets or issues have been linked in the PR description (or missing issue).
- [ ] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [ ] Unit tests have been added for the changes (if applicable).
- [ ] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

**Comment by lumixraku** [comment]
It's wried that scene.addObject  won't call markDirty of layer.

now transformByState dirty layer.

image

File: packages/docs-ui/src/services/selection/text-range.ts [hunk 1]
```diff
@@ -121,25 +121,23 @@ export class TextRange implements IDocRange {
...
                     setTimeout(() => {
                         this.deactivateStatic();
                     }, BLINK_ON);
                 }
             }
-        }, BLINK_OFF + BLINK_ON);
+        };
+
+        // wait for anchorShape created.
+        setTimeout(() => {
```
Comment (Jocs) [inline packages/docs-ui/src/services/selection/text-range.ts:136]: 这个能确保anchorShape 创建吗？

Comment (lumixraku) [reply packages/docs-ui/src/services/selection/text-range.ts:136]: 在这个场景可以😌

**Comment by lumixraku** [comment]
WeChatWorkScreenshot_e94b8265-5736-4c8b-aabc-9208e1a37d48


image

File: packages/docs-ui/src/services/selection/text-range.ts (hunk 1 above)
Comment (Jocs) [inline packages/docs-ui/src/services/selection/text-range.ts:136]: 有了下面 make dirty 是不是不用setTimeout 呢

File: packages/docs-ui/src/services/selection/text-range.ts [hunk 2]
```diff
@@ -121,25 +121,23 @@ export class TextRange implements IDocRange {
     }
 
     private _anchorBlink() {
-        setTimeout(() => {
```
Comment (Jocs) [inline packages/docs-ui/src/services/selection/text-range.ts:124]: 为啥删除这个呢？这个是为了 500ms 后隐藏

**Review by Jocs — APPROVED** [review]

**Pull request merged**